src/styles/header.scss

```diff
@@ -0,0 +1,61 @@
+@import 'variables';
+
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  padding: 0 $header-padding;
```

Comment:
The variable `$header-padding` should be defined in the `variables.scss` file or imported before use to avoid potential errors.